modules/sb-install-tsb-operator.adoc

```diff
@@ -15,13 +15,29 @@ The following procedure installs the {tsb-name} Operator using the
 web console.
 
 . Create a namespace.
+
 .. Navigate in the web console to *Administration* -> *Namespaces* and click *Create Namespace*.
-.. Enter `openshift-template-service-broker` in the *Name* field and click *Create*.
+
+.. Enter the following:
++
+--
+* `openshift-template-service-broker` in the *Name* field
+* `openshift.io/cluster-monitoring=true` in the *Labels* field
+--
++
+Then click *Create*.
 +
-NOTE: The namespace must start with `openshift-`.
+[NOTE]
+====
+The namespace must start with `openshift-`.
+====
+
 . Navigate to the *Operators* -> *OperatorHub* page. Verify that the `openshift-template-service-broker` project is selected.
+
```

Comment:
Is there a reason you added these? These shouldn't be necessary between steps unless you add text/code block afterward.

Reply:
Was just for readability / uniformity when there's plus sign continuations in the mix.